HLeithner/joomla-cms
# Add prepared statements for plg_extension-joomla

This PR has to be merged first https://github.com/joomla-framework/database/pull/156

### Summary of Changes
Updated SQL queries to prepared statements and made some cleanups around the queries.


### Testing Instructions
This plugin interacts with extension installation/removal/updates

* Install extensions
* Check for updates
* Update extensions
* Remove extensions



### Expected result
Nothing changed.
